static/src/javascripts/projects/common/modules/experiments/__tests__/ab-ophan.js

```diff
@@ -0,0 +1,96 @@
+// @flow
```

Comment:
Can we move this file to sit at the same directory level as the code under test?